### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by HoOngEe**
Update with Randomized Leader Election Proposal and Tenderand post

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,8 @@
 - [Consensus Compare: Tendermint BFT vs. EOS dPoS](https://blog.cosmos.network/consensus-compare-tendermint-bft-vs-eos-dpos-46c5bca7204b)
 - [List of projects in Cosmos & Tendermint Ecosystem](https://forum.cosmos.network/t/list-of-projects-in-cosmos-tendermint-ecosystem/243)
 - [The Most Comprehensive Guide to Tendermint Core](https://blockgeeks.com/guides/ultimate-guide-tendermint/)
+- [Randomized Leader Election Proposal](https://research.codechain.io/t/randomized-leader-election-using-vrf/17)
```
Comment (melekes) [inline README.md]: This is the original proposal, right? Don't think we need it here since the below post contains a link to it anyway.

```suggestion
```

**Review by melekes — COMMENTED** [review]
👍

**Comment by HoOngEe** [comment]
@melekes I cleaned up the commit and add non-English video resources. Recheck the pr, please.

**Review by melekes — APPROVED** [review]
💛

**Pull request merged**